src/action.ts

```diff
@@ -15,18 +26,45 @@ import { getTag } from "./utils.js";
 
 export async function performAction(
 	newMeta: Metafile,
+	decision: Decision,
 	searchee: Searchee,
 	tracker: string,
 	nonceOptions: NonceOptions
 ): Promise<ActionResult> {
-	const { action } = getRuntimeConfig();
+	const { action, linkDir } = getRuntimeConfig();
+
+	if (linkDir) {
```

Comment:
Why? The user will need to configure their watched torrents directory, but unless the depth is limited to 0 and safe matching is picked, using the same directory as the source directory won't work as it will match on torrents that don't fit the format of the saved ones.

Reply:
I think of Action.SAVE as less of a "put this into your deluge watch folder" and more of a "run autotorrent on cron" thing. Once we add deluge support (which is pretty high on the list) the remaining usecases for Action.SAVE are really more of an escape hatch/integration point than anything else

Reply:
i think we can actually probably just condition on `searchee.path` like the other spots

Reply:
but still, as it stands, with `Action.SAVE`, the output isn't going to contain enough information for the user to know if the underlying data is in linkDir or in dataDir

Reply:
Right now even perfect matches link, do you still think we shouldn't link for it? I suppose if the goal is to allow for hybrid usage this comment makes sense. I can gate linking behind inject, though some of the code will need a little rethinking, at the moment I wrote it with the linking assumption.

Reply:
I think we could limit max depth to 0 if Action.SAVE is enabled. Mainly putting this here as a reminder to myself when I'm lucid enough to dive into the code again